Cargo.toml

```diff
@@ -34,6 +34,7 @@ sha2 = { version = "0.10.8", optional = true }
 sha3 = "0.10.8"
 subtle = "2.5.0"
 thiserror = "1.0"
+zipf = { version = "7.0.1", optional = true }
```

Comment:
This is a bummer :( Is there any way to tell `cargo vet` that this is only used for "test-util" and should be ignored?

Reply:
No, but adding an exemption is easy, and we started off with that for some such dependencies. OTOH, it's only ~300 lines, it should be easy to re-audit it for safe-to-deploy.

Reply:
OK. I have absolutely no idea how this works, just let me know what I need to do.

Reply:
Thanks for #961.